lms/static/js/staff_debug_actions.js

```diff
@@ -4,7 +4,11 @@ var StaffDebug = (function() {
     /* global getCurrentUrl:true */
     var getURL = function(action) {
         var pathname = this.getCurrentUrl();
-        return pathname.substr(0, pathname.indexOf('/courseware')) + '/instructor/api/' + action;
+        var index = pathname.indexOf('/courseware');
+        if (index <= 0) {
+            index = pathname.indexOf('/', '/courses/'.length);
```

Comment:
Is the code trying to extract course_id?

Reply:
This function creates ` URL` form the current `URL` in specific format like `/courses/course-v1:edX+DemoX+mycourse/instructor/api`. This function was compatible with the course page and caused a bug on the instructor's tab.